### ISSUE TITLE
glm not in project?

### ISSUE THREAD

**Issue by coder-paullee** [Morozov-5F/glsdk#3]
what steps to build this project?

**Comment by coder-paullee** [comment]
what steps to build this project?

**Comment by Morozov-5F** [comment]
Hi @coder-paullee the build system is somewhat weird. In order to build it you need to run `lua make_distro.lua`. I've made that to work on the latest Fedora system at a time but I'm not sure whether it build or not right now. The script should download all the necessary dependencies.

**Comment by Morozov-5F** [comment]
Yeah, and I've left out glscene for now because I couldn't find a way to build the required parts of GLM with the modern compiler.
